app/assets/javascripts/controllers/middleware_deployment/middleware_deployment_controller.js

```diff
@@ -21,7 +23,8 @@ function MwAddDeploymentController($scope, $http, miqService) {
           miqService.miqFlash(result.data.status, result.data.msg);
         },
         function() { // error
-          miqService.miqFlash('error', 'Unable to deploy "' + data.runtimeName + '" on this server.');
+          miqService.miqFlash('error', 'Unable to deploy "' + data.runtimeName + '" on this server' +
+              (isGroupDeployment ? ' group' : '') +'.');
```

Comment:
The content of the `miqFlash()` here needs to be inside a `__()` so that it's correctly translated. Since this
is a dynamic string, you also need to use `sprintf(_("Whatever %s %s"), data..., data...)`